modules/banner/react/lib/Banner.tsx

```diff
@@ -20,9 +20,9 @@ export interface BannerProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
    */
   variant?: BannerVariant;
   /**
-   * Color theme of the banner
+   * Color of the banner based on the type of the error
    */
-  theme?: BannerTheme;
+  error?: BannerErrorType;
```

Comment:
Can you update this to use our generic `ErrorType` type? (https://github.com/Workday/canvas-kit/blob/master/modules/common/react/lib/types.ts#L22)

You can then remove `BannerTheme`/`BannerErrorType`